### PR TITLE
Page event show

### PR DESCRIPTION
**Pull request by Khaleddwm**
Ajout d'un extension twig
* Faire un composer install
* Yarn install

### CODE REVIEW

File: src/Entity/Event.php [hunk 1]
```diff
@@ -32,11 +32,21 @@ class Event
      */
     private $dateStart;
 
+    /**
+     * @ORM\Column(type="time")
+     */
+    private $dateTime;
+
```
Comment (RomainClair) [inline src/Entity/Event.php:39]: dateStart should probably be named startDate but what's the dateTime for ?

File: src/Form/EventType.php [hunk 2]
```diff
@@ -8,16 +8,24 @@
...
         $builder
             ->add('name', TextType::class)
-            ->add('description', TextType::class)
+            ->add('description', TextareaType::class)
             ->add('dateStart', DateType::class, [ 'widget' => 'single_text' ])
+            ->add('dateTime', TimeType::class, [
+                'placeholder' => [
+                    'hour' => 'Heure', 'minute' => 'Minute'
+                ]
+            ])
```
Comment (RomainClair) [inline src/Form/EventType.php:26]: A DateTime object store a complete date with hours and minute.

File: src/Entity/Event.php (hunk 1 above)
Comment (Khaleddwm) [reply src/Entity/Event.php]: Ok the changes are made ...

File: src/Form/EventType.php (hunk 2 above)
Comment (Khaleddwm) [reply src/Form/EventType.php]: Ok the changes are made ...

**Pull request merged**